app/routes/v2/standards/index.js

```diff
@@ -44,3 +44,22 @@ module.exports = [{
     }
   }
 }]
+
+const getAllStandards = async (request, error) => {
```

Comment:
This would be be better either as its own Module or as part of the ViewModel. Its just good to keep the routes as clean and light weight as possible